README.md

```diff
@@ -54,6 +54,11 @@ sources:
   rev: master
   scripts:
   - chmod a+x truffleHog.py
+- name: fontawesome
+  repo: https://github.com/FortAwesome/Font-Awesome
+  rev: master
+  sparse_paths:
```

Comment:
Let's move this up a line to match the generated configuration.

Reply:
done